api/src/main/java/org/pmiops/workbench/actionaudit/ActionAuditQueryServiceImpl.java

```diff
@@ -117,38 +118,20 @@ public UserAuditLogQueryResponse queryEventsForUser(
     return new UserAuditLogQueryResponse()
         .actions(auditLogEntryMapper.logEntriesToActions(logEntries))
         .logEntries(logEntries)
-        .query(getReplacedQueryText(queryJobConfiguration))
+        .query(BQParameterUtil.getReplacedQueryText(queryJobConfiguration))
         .userDatabaseId(userDatabaseId);
   }
 
   private ImmutableMap.Builder<String, QueryParameterValue> getNamedParameterMapBuilder(
-      long limit, DateTime after, DateTime before) {
+      long limit, Instant after, Instant before) {
+    final Instant afterPartitionTime = after.minus(PARTITION_BUFFER);
+    final Instant beforePartitionTime = before.plus(PARTITION_BUFFER);
```

Comment:
I mean that I don't think adding one day to "before" is necessary.

Reply:
Oh, the timestamp I send to StackDriver is generally earlier than the ingestion time (which is what counts). So if I have an event at 23:59:30, it might have an ingestion timestamp of 00:00:30 on the next day. But you're right that that's fixable by moving before to the right.

Reply:
OK - safer to keep this then.  thanks!